### DISCUSSION TITLE
Hubble's parameter vs Hubble's constant

### DISCUSSION THREAD

**Discussion by mikehelland** [a-cosmology-group/acg#234] [Cosmology Models]
This applies to FLRW models, which LCDM is based on.

FLRW models have a Hubble parameter, $H$, and a Hubble constant, $H_0$, related by this:

$$H = H_0 [\Omega_M (1+z)^3 + \Omega_\Lambda]^{1/2}$$ 

Hubble's constant, $H_0$ is the expansion rate today.

In a de Sitter model, aka pure dark energy, aka exponentially expanding model you have:

$$\Omega_M = 0.0$$

$$\Omega_\Lambda = 1.0$$ 

$$[\Omega_M (1+z)^3 + \Omega_\Lambda] = 1$$

$$H = H_0$$

The expansion rate never changes.

In LCDM you have:

$$\Omega_M = 0.3$$

$$\Omega_\Lambda = 0.7$$ 

Which gives us something like this:

![image](https://github.com/a-cosmology-group/acg/assets/19311107/16a878e0-66e1-4f21-b40c-7bd58e3b6926)

It was larger in the past, and is settles on a value around 55 in the infinite future.

You can use this to try your own parameters:

https://mikehelland.github.io/hubbles-law/other/expansionrate/expansionrate.htm

To try to address a question by @ExpEarth, you can play with the parameters like this:

$$\Omega_M = -0.1$$

$$\Omega_\Lambda = 1.1$$ 

And the Hubble parameter will be lower in the past, rather than higher. I tried to compute luminosity distances that model, but it started complaining about not real numbers.

**Comment by HanDeBruijn** [comment]
As is explained in the section [Hubble Parameter](http://www.alternatievewiskunde.nl/cosmology/Flandern.htm) at my website, the favorite model associated with the [Variable Mass Theory](https://www.youtube.com/watch?v=T7jVviVBzKs) is a **de Sitter** like cosmological model with accompanying constant Hubble parameter and an ( [equivalent](http://www.alternatievewiskunde.nl/cosmology/lengte.htm) ) exponential expansion rate that never changes.
A non-empty universe, filled _with_ common mass and _without_ dark energy.
